djstripe/models/core.py

```diff
@@ -823,8 +824,16 @@ def create(cls, subscriber, idempotency_key=None, stripe_account=None):
         if subscriber_key not in ("", None):
             metadata[subscriber_key] = subscriber.pk
 
+        if hasattr(subscriber, 'name') and subscriber.name:
+            # if subscriber table has name column use it as customer name
+            name = subscriber.name
+        else:
+            # use email's username as customer name
+            name = re.sub('[_.-]', ' ', subscriber.email.split('@')[0]).title()
+
```

Comment:
There is a more pythonic way of doing this:

```py
try:
    # if subscriber table has name column use it as customer name
    name = subscriber.name
except AttributeError:
    pass
```


Let's not create `name` automatically as `subscribers` can be organisations with just 1 org email.